src/app/components/oeb-dialog.component.ts

```diff
@@ -23,9 +23,10 @@ import { NgClass, NgIf } from '@angular/common';
 		<div
 			class="tw-px-4 tw-py-6"
 			[ngClass]="{
-				' tw-border-red': variant === 'danger',
-				' tw-border-link': variant === 'info',
-				'tw-border-purple': variant === 'default'
+				'!tw-border-red !tw-bg-white !tw-border-4 !tw-border-solid': variant === 'danger',
+				'!tw-border-link !tw-bg-white !tw-border-4 !tw-border-solid': variant === 'info',
+				'!tw-border-green !tw-bg-green': variant === 'success',
```

Comment:
Is it necessary to use the !important variants here?
As a [best practice](https://developer.mozilla.org/en-US/docs/Web/CSS/important#best_practices), !important should be prevented and since this is already using conditional assignment of classes, this should be necessary here, right?

If it is necessary, I think this would deserve a comment why.